### PR TITLE
Resource packs

### PR DESCRIPTION
**Pull request by AppleDevelops**
This PR adds resource packs, note that atm you cannot join the server and I am working on that.

### CODE REVIEW

**Comment by Driesboy** [comment]
a player that is banned will download the tecturepack

**Comment by ISRAPIL** [comment]
It not bad problem... @Driesboy

**Comment by Prisane** [comment]
now working?

**Comment by Driesboy** [comment]
its a bad internet problem!

File: src/pocketmine/resources/tesseract_eng.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ anticheat:
  #Recommended that it stays "false"
  allow-instabreak: false
  #This setting decides if flying will be detected/tracked or not
- #Recommended it is set to "true", but set it to "false" if it bugs out players that do not use modifiers
+ #Recommended it is set to "true", but sst it to "false" if it bugs out players that do not use modifiers
```
Comment (Driesboy) [inline src/pocketmine/resources/tesseract_eng.yml]: sst?

**Comment by caspervanneck** [comment]
@AppleDevelops  I cant join

**Comment by SuperMaXAleX** [comment]
God, I created an another branch to work on it .-. I never said to merge it into master branch !

**Comment by SuperMaXAleX** [comment]
whut

**Comment by SuperMaXAleX** [comment]
I were working on it .-------------.

**Comment by AnkitM252** [comment]
Restored!!

**Comment by SuperMaXAleX** [comment]
k so github is glitching

**Comment by AnkitM252** [comment]
No, I removed it! I think its completed..

**Comment by NFGamerMC** [comment]
Stop, it is just going to be in master branch!

**Pull request merged**